client/pages/auth-modal.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import AuthForm from '../components/auth-form';
+
+export default class AuthModal extends React.Component {
+  render() {
+    return (
+      <>
+      <div className='modal'>
+          <AuthForm onAuthClick={this.props.onAuthClick}/>
+      </div>
+      </>
```

Comment:
You can remove this react fragment

Reply:
<img width="165" alt="Screen Shot 2022-09-16 at 9 59 34 AM" src="https://user-images.githubusercontent.com/106217355/190690997-b605eee1-ef00-473c-8019-fff9478230db.png">